lib/google/serverless/exec.rb

```diff
@@ -376,7 +376,8 @@ class << self
         #     for App Engine Flexible and Cloud Run).
         # @param gcs_log_dir [String,nil] GCS bucket name of the cloud build log
         #     when strategy is "cloud_build". (ex. "gs://BUCKET-NAME/FOLDER-NAME")
-        #
+        # @param product [Symbol] The serverless product to use. If omitted, defaults to
+        #     the value returned by {Google::Serverless::Exec.default_product}
```

Comment:
```suggestion
        #     the value returned by {Google::Serverless::Exec#default_product}
```

To reference a normal method, use `#`. The `.` character is used to reference _class methods_.

Reply:
Oh! This is new knowledge to me! Thanks Daniel